core/editor/assets/js/editor-loader.js

```diff
@@ -0,0 +1,6 @@
+if ( 2 === globalThis?.elementorEditorLoaderConfig?.version ) {
```

Comment:
extract to multiple entries one for each version, so we won't have those conditions everywhere